From 777ac896daaffeaa2fac2bdb424a96def7409a4b Mon Sep 17 00:00:00 2001
From: Nathan Rossi <nathan@nathanrossi.com>
Date: Wed, 18 Oct 2017 21:29:47 +1000
Subject: [PATCH] arm64: zynqmp: Setup partid for QEMU to match silicon

During board late init the environment is 'setup' to set the partid to 0
for QEMU. Change this so that QEMU targets behave just like silicon and
emulation targets such that partid is set to auto.

Signed-off-by: Nathan Rossi <nathan@nathanrossi.com>
Upstream-Status: Submitted [sent to git@xilinx.com]
---
 board/xilinx/zynqmp/zynqmp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/xilinx/zynqmp/zynqmp.c b/board/xilinx/zynqmp/zynqmp.c
index fd80844df6..2df66a4b75 100644
--- a/board/xilinx/zynqmp/zynqmp.c
+++ b/board/xilinx/zynqmp/zynqmp.c
@@ -294,9 +294,9 @@ int board_late_init(void)
 		setenv("setup", "setenv baudrate 4800 && setenv bootcmd run veloce");
 	case ZYNQMP_CSU_VERSION_EP108:
 	case ZYNQMP_CSU_VERSION_SILICON:
+	case ZYNQMP_CSU_VERSION_QEMU:
 		setenv("setup", "setenv partid auto");
 		break;
-	case ZYNQMP_CSU_VERSION_QEMU:
 	default:
 		setenv("setup", "setenv partid 0");
 	}
-- 
2.15.0

